python-sdk/src/astro/sql/operators/dataframe.py

```diff
@@ -262,10 +265,29 @@ def get_openlineage_facets_on_complete(self, task_instance):  # skipcq: PYL-W061
 
         run_facets: dict[str, BaseFacet] = {}
         job_facets: dict[str, BaseFacet] = {}
+        collect_source = os.environ.get("OPENLINEAGE_AIRFLOW_DISABLE_SOURCE_CODE", "True")
```

Comment:
Can we add this to the `settings.py` in root like other environment variables? Also we should be doing `conf.get()` from `from airflow.configuration import conf` as there are various ways in airflow to get env.

Reply:
moved to settings